runtime/src/accounts_db.rs

```diff
@@ -1095,14 +1182,35 @@ impl AccountsDB {
         }
     }
 
-    pub fn xor_in_hash_state(&self, slot_id: Slot, hash: BankHash, stats: &BankHashStats) {
-        let mut bank_hashes = self.bank_hashes.write().unwrap();
-        let bank_hash = bank_hashes
-            .entry(slot_id)
-            .or_insert_with(BankHashInfo::default);
-        bank_hash.hash.xor(hash);
-
-        bank_hash.stats.merge(stats);
+    pub fn get_accounts_delta_hash(&self, slot_id: Slot) -> Hash {
+        let mut scan = Measure::start("scan");
+        let mut accumulator: Vec<HashMap<Pubkey, (u64, Hash)>> = self.scan_account_storage(
```

Comment:
Related to [this](https://github.com/solana-labs/solana/pull/8295/files#r382658701), this could be made BTreeMap for general faster sort?